### DISCUSSION TITLE
Form Creation In Pimcore

### DISCUSSION THREAD

**Discussion by mangaigsm** [pimcore/pimcore#12906] [Q&A / Support]
Hi,
Kindly Share the Steps to create a Form In Pimcore.
My Requirement :
1. Need to Create a Form with Textboxes
2. Read Value From the Form
3. While submitting the form, it should create a Data Object in the specified Data Object Folder.
Kindly Share your ideas to Achieve my Requirement
Thanks.

**Comment by KKCB** [comment]
You can achieve this with the FormBuilder Plugin: https://github.com/dachcom-digital/pimcore-formbuilder

Have a look at the "object channel" output workflow: https://github.com/dachcom-digital/pimcore-formbuilder/blob/master/docs/OutputWorkflow/11_ObjectChannel.md

**Comment by dpfaffenbauer** [comment]
or use Symfony Form directly: https://symfony.com/doc/current/forms.html

**Comment by BlackbitDevs** [comment]
It is not clear if you want to create a frontend application, then the Formbuilder plugin may be a valid option.

In case you want a form in the Pimcore backend for better UX when creating new objects, then please have a look at https://www.youtube.com/watch?v=0bjFNQfrzlw&list=PL4-QRNfdsdKIfzQIP-c9hRruXf0r48fjt&index=11&t=320s - with this you can create a form in the Pimcore backend without any programming (but you can add programming logic if you want to).
